.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: minimal
+language: cpp
```

Comment:
@junr03 oh, indeed, you explained it there, sorry, I should have read it first!

However, the issue remains - `language: minimal` is not recognized by `travis lint`, which fails because of it... but because it doesn't set exit code on errors by default, it doesn't break build.

Would you mind adding `language: minimal` to [travis.rb](https://github.com/travis-ci/travis.rb) (presumably, that's the correct repo) or do we want to keep ignoring `travis lint` failures?